enterprise/static/enterprise/js/grant_data_sharing_permissions.js

```diff
@@ -47,49 +26,29 @@ function loadConsentPage() {
         $("#data-consent-checkbox").attr("checked", false);
         $("#data-sharing").submit();
     });
-    $("#review-policy-link").click(function (event) {
+    $reviewPolicyLink.click(function (event) {
         event.stopPropagation();
-        hideConsentConfirmationModal();
         togglePolicy(true);
+        $("#consent-confirmation-modal").modal('hide');
         $("#consent-policy-dropdown-bar").focus();
     });
-    $("#consent-confirmation-modal").click(function (event) {
-        hideConsentConfirmationModal();
-        $("#failure-link").focus();
-    });
     $("#data-consent-checkbox").change(function (event) {
         $("#consent-button").attr("disabled", !(this.checked));
     });
-    $(document).keydown(function (event) {
-        if ((event.keyCode === 27) && ($("#consent-confirmation-modal").is(":visible"))) {
-            // If the modal is shown, and the ESC key is pressed, hide the modal.
-            hideConsentConfirmationModal();
-            $("#failure-link").focus();
-            event.preventDefault();
-        }
-    });
-    $("#review-policy-link").keydown(function (event) {
-        if (event.keyCode == 9 && !event.shiftKey){
+    $reviewPolicyLink.keydown(function (event) {
+        if (event.keyCode === 9 && !event.shiftKey){
             // Catch the tab keydown event when leaving the last control in the modal,
             // and move the focus to the first control in the modal.
             event.preventDefault();
             $("#modal-close-button").focus();
         }
     });
-    $("#modal-close-button").keydown(function (event) {
```

Comment:
I see we're still manually handling the forward wraparound to the top of the modal; why are we removing the backward wraparound to the end of the modal?

Reply:
You're right. Not sure why I erased this. Thanks for pointing this out!